python_modules/libraries/dagster-fivetran/dagster_fivetran/translator.py

```diff
@@ -24,12 +24,17 @@
 class FivetranConnectorTableProps(NamedTuple):
     table: str
     connector_id: str
-    name: str
+    connector_name: str
     connector_url: str
+    destination_id: Optional[str]
```

Comment:
Must be optional for the legacy integration - we don't have access to the destination ID when creating the props in the legacy integration, and making that change is not worth it.